HW03/Dmitry_Susha/dmitry_susha_hw_03_t_02.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class Homework3
+  def task2(logs)
+    result = []
```

Comment:
Split your log here, and do all the necessary work by parsing in an iterator over the split data

```suggestion
    lines = logs.split
    
    lines.select { |line| what you want to select }. map |line| do
      selected lines manipulation 
    end
```